src/common/transformations/src/transformations/mlir/mlir_op.cpp

```diff
@@ -64,9 +72,9 @@ void prepareMLIRKernelWithoutWrapper(mlir::OwningOpRef<mlir::ModuleOp>& module)
     // A set of default passes that lower any input IR to LLVM
     PassManager pm(module->getContext());
 
-#if 0  // TODO: if TPP is available
+#if TPP_MLIR && false  // FIXME: Still cannot enable it: undefined reference to mlir::createConvertOpenMPToLLVMPass(), mlir::createParallelLoopToGpuPass() etc.
```

Comment:
@rengolin, looks like we need to add more libraries in this list: https://github.com/slyalin/openvino/blob/mlir/src/cmake/openvino.cmake#L45-L59. But I don't understand how quickly find out which ones. Could you help?

Reply:
They should be in the MLIR paths, they're set by CMake via tricks like `add_llvm_executable` ([Link](https://github.com/plaidml/tpp-mlir/blob/295409bfbb271f9d5ad08096213ca7efe7210491/tools/tpp-opt/CMakeLists.txt#L24)) `and add_mlir_library` ([link](https://github.com/plaidml/tpp-mlir/blob/295409bfbb271f9d5ad08096213ca7efe7210491/lib/TPP/CMakeLists.txt#L12C1-L12C17))

Reply:
These macros cannot be used in openvino I suppose, as they are adding a new library or executable target, but we already have the library. Did you mean I can reverse-engineer those macros and understand how to get the required libraries? Is there a more regular way to do that without adding a new target?

Reply:
Either way. If you reverse engineer, it should be just a matter of finding the header/lib paths for MLIR and add those to the targets you need.

Reply:
OK, I found a way to build it successfully. Just using `MLIR_ALL_LIBS` property. But enabling it in this place fails in runtime with segfault.

Reply:
Where in the code is the segfault? TPP-MLIR? MLIR? LLVM?